ecosystem/python/sdk/examples/multisig.py

```diff
@@ -0,0 +1,483 @@
+import subprocess
+import time
+
+from aptos_sdk.account import Account, RotationProofChallenge
+from aptos_sdk.account_address import AccountAddress
+from aptos_sdk.authenticator import Authenticator, MultiEd25519Authenticator
+from aptos_sdk.bcs import Serializer
+from aptos_sdk.client import FaucetClient, RestClient
+from aptos_sdk.ed25519 import MultiEd25519PublicKey, MultiEd25519Signature
+from aptos_sdk.transactions import (EntryFunction, RawTransaction, Script,
+                                    ScriptArgument, SignedTransaction,
+                                    TransactionArgument, TransactionPayload)
+from aptos_sdk.type_tag import StructTag, TypeTag
+from common import FAUCET_URL, NODE_URL
+
+wait_for_user = True
```

Comment:
not sure this adds much value, can we remove it?

Reply:
@davidiw removed.